test-resources/bgp/expected_full_report.txt

```diff
@@ -26,13 +26,13 @@ Authorizations which may be too permissive:
 		10.0.0.0/22 => 64497
 		10.0.0.0/24 => 64496
 
-AS0 Authorizations disallowing announcements for prefixes
+AS0 ROA configurations disallowing announcements for prefixes
 
-	Definition: 10.0.4.0/24 => 0
+	Configuration: 10.0.4.0/24 => 0
 
-AS0 Authorization which are made redundant by authorizations for the prefix from real ASNs
+AS0 Authorization which are made redundant by configuration(s) for the prefix from real ASNs
```

Comment:
Two lines up `AS0 Authorizations` was changed to `AS0 ROA configurations`, why then on this line is `AS0 Authorization` left unchanged?

Reply:
ack. fixed